Deployment/Jenkins/master/Master-Ec2-Instance.groovy

```diff
@@ -0,0 +1,281 @@
+pipeline {
+
+agent any
```

Comment:
Style-comment: `pipeline {}` is an containing-object that the `agent`, `options {}` etc. are nested-within. You'll want to indent everything between the `pipeline {` and final `}` by four spaces.

Reply:
Done